.eslintrc.i18n.js

```diff
@@ -0,0 +1,13 @@
+const path = require("path");
+
+module.exports = {
+    extends: ["plugin:i18n-json/recommended"],
+    rules: {
+        "i18n-json/identical-keys": [
+            "warn",
+            {
+                filePath: path.resolve("./i18n/en.json"),
```

Comment:
Can you pass ko.json/all files in that directory?

Reply:
This is the reference file to compare against

Reply:
This is the reference file to compare against